### DISCUSSION TITLE
Add the following options: treasure pool icon above the Party List, MP cost above Party List (when casting from menu), and keep Party List at full size at all times,

### DISCUSSION THREAD

**Discussion by AzrielltheWise** [tirem/XIUI#13] [Ideas]
First, I love HXUI.  After discovering it and playing around with it for a bit, it was clear that this is how I want to play XI from now on.  However, there are a few small issues preventing me from switching over full time.

My preference is to place the HXUI Party List over the default FFXI UI Party List and then change the UI element for Party List to a solid color to fully obscure the default UI element.  This looks great and works great, but I quickly realized that you can no longer see the treasure pool icon (the icon that indicates that loot is in the treasure pool).  Next, I realized that I can no longer see how much a spell costs if I am casting from the menu.  It would be great if there was an HXUI element that showed both of these items (preferably these would be affixed to Party List and disappear when not needed, similar to how the default UI works, but even movable elements similar to the Gil Tracker and Inventory Tracker would work).

An alternate solution to the above problems might be to let us shrink Party List to be the same height as the default UI Party List, so that these two UI elements are not obscured.  I'm not sure how good that would look though.

Finally, I would greatly appreciate an option to keep the Party List at full size the entire time, even when you aren't in a full 6/6 party.  If I am in a less-than-full party, Party List looks weird floating above  the default Party List instead of covering it.  I prefer static UI elements (e.g my chat boxes don't resize and I have my macros set to static positioning), so I think this would be a nice option to have.

Thanks for the great addon.  HXUI is one of my favorite new additions to Ashita4/Horizon.  FFXI has never looked or performed so well and UI addons like HXUI are a huge reason why.

**Comment by FurizuTenshi** [comment]
an addon to this post, Recast timer of the selected spell, just anything that would be hidden down there really

**Reply by AzrielltheWise** [reply]
Yeah.  I just noticed the Cast Lots results are hidden as well with this setup.  A lot of stuff I just take for granted, apparently.  Hopefully a solution can be implemented because I am absolutely loving using this in PTs, especially as a healer.

Edit:  Conquest Points are also covered on the Region Map, so maybe an option to hide HXUI when Map, Region Map, and/or menus are open would be handy.  I played around with another UI mod before finding HXUI and I believe there was an option along these lines.

**Comment by AzrielltheWise** [comment]
Another issue with putting HXUI's Party List on top of the vanilla party window:  When rearranging the party list, the vanilla window changes to display party member jobs. 

I would suggest an option to display jobs on HXUI's Party List.
